CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 1. [21519](https://github.com/influxdata/influxdb/pull/21519): Upgrade Flux to v0.117.0
 1. [21519](https://github.com/influxdata/influxdb/pull/21519): Optimize `table.fill()` execution within Flux aggregate windows.
+1. [21564](https://github.com/influxdata/influxdb/pull/21564): Upgrade UI to v2.0.7
```

Comment:
Is there an easy way for users to find what this entails (i.e. a CHANGELOG, or notes on the public docs site)? If not, could we instead summarize the changes made between UI v2.0.5->v2.0.7?

Reply:
There are release notes for the v2.0.7 on the release in the `ui` folder similar to the Flux release notes - I updated the CHANGELOG here to include a link to them.